src/ol_social_auth/backends.py

```diff
@@ -3,10 +3,10 @@
 from social_core.backends.oauth import BaseOAuth2
 
 
-class MITxProOAuth2(BaseOAuth2):
+class OLOAuth2(BaseOAuth2):
     """MIT xPro social auth backend"""
```

Comment:
class docstring should also be updated
e.g; "Open Learning social auth backend"

Similarly review and update file-level description as well